superdesk/text_checkers/ai/__init__.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2020 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from superdesk.resource import Resource
+from superdesk.services import BaseService
+from superdesk.errors import SuperdeskApiError
+from .. import tools
+from .base import registered_ai_services, AIServiceBase
+import superdesk
+
+AUTO_IMPORT = True
+
+
+class AIResource(Resource):
+    schema = {
+        'service': {
+            'type': 'string',
+            'required': True,
```

Comment:
yes it's only iMatrics, I've put it in a more genertic endpoint as I suspect we'll have more machine learning service in the future (it's trendy nowadays).
I'll add the allowed as you suggest.

Reply:
@ride90 thinking twice about it, if I limit names with `allowed` we're loosing new services detection (we just have to create a new service and it will be automatically added to the endpoint), so I think using `allowed` is not a good idea at the end.

Reply:
you can do something like in [ingest](https://github.com/superdesk/superdesk-core/blob/develop/superdesk/io/ingest_provider_model.py#L71)

Reply:
oh indeed, haven't thought of doing it dynamically, good point I'll update.

Reply:
done in latest update